incubator/hnc/pkg/testutils/testutils.go

```diff
@@ -46,25 +46,50 @@ func RunShouldContain(substr string, seconds float64, cmdln ...string) {
 
 func RunShouldContainMultiple(substrs []string, seconds float64, cmdln ...string) {
 	Eventually(func() error {
-		stdout, err := RunCommand(cmdln...)
-		if err != nil {
+		return runShouldContainMultiple(substrs, cmdln...)
+	}, seconds).Should(Succeed())
+}
+
+func RunErrorShouldContain(substr string, seconds float64, cmdln ...string) {
+	RunErrorShouldContainMultiple([]string{substr}, seconds, cmdln...)
+}
+
+func RunErrorShouldContainMultiple(substrs []string, seconds float64, cmdln ...string) {
+	Eventually(func() error {
+		err := runShouldContainMultiple(substrs, cmdln...)
+		GinkgoT().Log("Error from: RunErrorShouldContainMultiple ", err)
+		if containAll([]string {"Expecting: ", " but get: "}, err.Error()) {
 			return err
 		}
+		return nil
+	}, seconds).Should(Succeed())
+}
 
-		allContained := true
-		for _, substr := range substrs {
-			if strings.Contains(stdout, substr) == false {
-				allContained = false
-				break
-			}
+func runShouldContainMultiple(substrs []string, cmdln ...string) error {
+		stdout, err := RunCommand(cmdln...)	
+		GinkgoT().Log("Output: ", stdout)
+		if err != nil {
+			return err
 		}
 
+		allContained := containAll(substrs, stdout)
+
 		if allContained == false {
```

Comment:
These two lines can now be combined after the change (into ` if !containAll(substrs, stdout){}`)